demo/api/src/importer/data-streams/data-source.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum DataSources {
```

Comment:
Please use the singular DataSource here

Reply:
why do we need this in the first place? it seems it's not really used, and we do have different classes for different data-sources...

Reply:
True, we actually don't need this